serverless.yml

```diff
@@ -78,6 +78,10 @@ provider:
         - "iam:DeleteGroupPolicy"
         - "iam:DeletePolicyVersion"
       Resource: "*"
+    - Effect: "Allow"
+      Action:
+        - "lambda:Invoke*"
+      Resource: "*"
```

Comment:
Could this be `${env:EXTERNAL_LAMBDA}` as well?

Reply:
Updated